pkg/engine/engine.go

```diff
@@ -115,6 +115,8 @@ func (e *Engine) Render(chrt *chart.Chart, values chartutil.Values) (map[string]
 type renderable struct {
 	// tpl is the current template.
 	tpl string
+	// path is the path of chart
+	path string
```

Comment:
Yes, we can extract a logic path from `basePath`. We can do it in next `rebase`.